holmes/config.py

```diff
@@ -431,6 +434,33 @@ def load_custom_toolsets_config(self) -> list[ToolsetYamlFromConfig]:
                 description: "Perform a curl request to example.com using variables"
                 command: "curl -X GET '{{api_endpoint}}?query={{ query_param }}' "
         """
+        loaded_toolsets = []
+        for custom_path in self.custom_toolsets:
+            if os.path.isfile(custom_path):
```

Comment:
Please extract it to method and reuse also in loading of CUSTOM_TOOLSET_LOCATION